### ISSUE TITLE
Unable to instantiate fragment com.reactnativestripesdk.PaymentLauncherFragment

### ISSUE THREAD

**Issue by iamgirya** [flutter-stripe/flutter_stripe#1473]
We use Stripe for payment. There was a task to add affirm. While working inside the affirm's page in browser, the application may crash at a random moment with the error "could not find Fragment constructor".

To reproduce:
1) get paymentIntentClientSecret
2) use 
```
await Stripe.instance.confirm Payment(
  paymentIntentClientSecret: paymentIntentClientSecret,
  data: const Payment Method Params.affirm(
    paymentMethodData: Payment Method Data(),
  ),
);
```
3) Wait for a while on the confirmation registration screen

Flutter 3.3.5
Android 10
flutter_stripe-9.5.0+1

I noticed that with each crash, the application becomes more stable and crashes after a longer period of time, as long as the application stops crashing altogether.
I found the same problem in these issues: https://github.com/flutter-stripe/flutter_stripe/issues/729 & https://github.com/stripe/stripe-react-native/issues/930
The method described in https://github.com/flutter-stripe/flutter_stripe/issues/729#issuecomment-1128634820 it has no effect.

The recurring part of the error:
```
Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.droppingbuy.buyer/com.droppingbuy.buyer.MainActivity}: androidx.fragment.app.Fragment$InstantiationException: Unable to instantiate fragment com.reactnativestripesdk.PaymentLauncherFragment: could not find Fragment constructor
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3433)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3572)
       at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
       at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:140)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2097)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:264)
       at android.app.ActivityThread.main(ActivityThread.java:7663)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:980)

Caused by androidx.fragment.app.Fragment$InstantiationException: Unable to instantiate fragment com.reactnativestripesdk.PaymentLauncherFragment: could not find Fragment constructor
       at androidx.fragment.app.Fragment.instantiate(Fragment.java:678)
       at androidx.fragment.app.FragmentContainer.instantiate(FragmentContainer.java:57)
       at androidx.fragment.app.FragmentManager$3.instantiate(FragmentManager.java:507)
       at androidx.fragment.app.FragmentState.instantiate(FragmentState.java:81)
       at androidx.fragment.app.FragmentStateManager.<init>(FragmentStateManager.java:85)
       at androidx.fragment.app.FragmentManager.restoreSaveStateInternal(FragmentManager.java:2505)
       at androidx.fragment.app.FragmentManager.attachController(FragmentManager.java:2665)
       at androidx.fragment.app.FragmentController.attachHost(FragmentController.java:117)
       at androidx.fragment.app.FragmentActivity.lambda$init$3$androidx-fragment-app-FragmentActivity(FragmentActivity.java:140)
       at androidx.fragment.app.FragmentActivity$$ExternalSyntheticLambda0.onContextAvailable(:2)
       at androidx.activity.contextaware.ContextAwareHelper.dispatchOnContextAvailable(ContextAwareHelper.kt:84)
       at androidx.activity.ComponentActivity.onCreate(ComponentActivity.java:378)
       at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:217)
       at io.flutter.embedding.android.FlutterFragmentActivity.onCreate(FlutterFragmentActivity.java:279)
       at android.app.Activity.performCreate(Activity.java:7805)
       at android.app.Activity.performCreate(Activity.java:7794)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1306)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3403)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3572)
       at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
       at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:140)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2097)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:264)
       at android.app.ActivityThread.main(ActivityThread.java:7663)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:980)

Caused by java.lang.NoSuchMethodException: com.reactnativestripesdk.PaymentLauncherFragment.<init> []
       at java.lang.Class.getConstructor0(Class.java:2332)
       at java.lang.Class.getConstructor(Class.java:1728)
       at androidx.fragment.app.Fragment.instantiate(Fragment.java:663)
       at androidx.fragment.app.FragmentContainer.instantiate(FragmentContainer.java:57)
       at androidx.fragment.app.FragmentManager$3.instantiate(FragmentManager.java:507)
       at androidx.fragment.app.FragmentState.instantiate(FragmentState.java:81)
       at androidx.fragment.app.FragmentStateManager.<init>(FragmentStateManager.java:85)
       at androidx.fragment.app.FragmentManager.restoreSaveStateInternal(FragmentManager.java:2505)
       at androidx.fragment.app.FragmentManager.attachController(FragmentManager.java:2665)
       at androidx.fragment.app.FragmentController.attachHost(FragmentController.java:117)
       at androidx.fragment.app.FragmentActivity.lambda$init$3$androidx-fragment-app-FragmentActivity(FragmentActivity.java:140)
       at androidx.fragment.app.FragmentActivity$$ExternalSyntheticLambda0.onContextAvailable(:2)
       at androidx.activity.contextaware.ContextAwareHelper.dispatchOnContextAvailable(ContextAwareHelper.kt:84)
       at androidx.activity.ComponentActivity.onCreate(ComponentActivity.java:378)
       at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:217)
       at io.flutter.embedding.android.FlutterFragmentActivity.onCreate(FlutterFragmentActivity.java:279)
       at android.app.Activity.performCreate(Activity.java:7805)
       at android.app.Activity.performCreate(Activity.java:7794)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1306)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3403)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3572)
       at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
       at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:140)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2097)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:264)
       at android.app.ActivityThread.main(ActivityThread.java:7663)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:980)
```

**Comment by remonh87** [comment]
this should be fixed in version 9.60

**Comment by iamgirya** [comment]
Yeap
I fixed it like this
![image](https://github.com/flutter-stripe/flutter_stripe/assets/84345515/f9ae55ee-3f29-4029-880c-3428e1d6bc8e)
And placed null safety signs in class.
Not sure if this is correct, but it works
